Automattic/wp-calypso
# Checkout: update ToS block for gift purchases

Much of our "Terms of Service" area in Checkout is crafted for the subscription owner. This removes the parts that are unnecessary for gifting a subscription renewal.

From the discussion on the design post (p9Jlb4-5Ae-p2#comment-6266), it seems that our standard ToS is sufficient for a gift purchase. I also opted to include the standard refund policy notice (refunds are available via support interaction).

I'm hiding the following because we can currently only gift renewals of domains and plans:
- domain registration agreement
- domain SSL requirement
- ~~concierge refund policy (we should maybe remove this)~~ removed - we don't sell new ones
- bundled domain notice
- Titan ToS
- Plugin ToS

| Before      | After |
| ----------- | ----------- |
| <img width="565" alt="Screen Shot 2022-11-16 at 1 49 48 PM" src="https://user-images.githubusercontent.com/942359/202309907-07ceee25-8873-47f4-a532-2342b4accb8a.png"> | <img width="576" alt="Screen Shot 2022-11-16 at 5 35 07 PM" src="https://user-images.githubusercontent.com/942359/202309942-6348616a-bbd2-4269-8490-c36afb27ccd9.png"> |

Fixes: https://github.com/Automattic/payments-shilling/issues/1196

**To test**
- add a gift renewal to your cart (or visit http://calypso.localhost:3000/checkout/personal-bundle/gift/1026184)
- verify that the ToS matches the screenshot
- don't check out (it makes testing easier) and empty your cart
- add a normal purchase to your cart (plan, plugin, add-on, domain, or any combination thereof)
- verify that the applicable ToS items are shown
